cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -8,7 +8,9 @@ toc::[]
 
 // The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
 
-You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. 
+
+//You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
```

Comment:
I'm guessing this text is describing Red Hat's [OpenShift Connector](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector). If so, what do you think about: 
```suggestion
You can also install the OpenShift Connector extension for Microsoft Visual Studio Code, which uses both the `{odo-title}` and the `oc` binaries to interact with your {product-title} cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
```

Reply:
Use {product-title} to refer to OCP.

Reply:
^^ This is the only non-nit.

Reply:
That paragraph is commented out entirely, as the functionality has been removed upstream docs. I will delete the content once an SME confirms that it is ok to remove downstream.

If it comes back in, I will make the changes indicated.

Reply:
Better to complete the SME review task before the peer review task.

Reply:
In a perfect world, yes. I am waiting on SME reviews for a lot of content and had hoped that putting the cart before the horse and asking for peer review in the mean-time might speed things up...